lib/Conversion/TorchToTcp/Elementwise.cpp

```diff
@@ -440,23 +335,64 @@ class ConvertAtenReluOp : public OpConversionPattern<AtenReluOp> {
   }
 };
 
+class ConvertAtenAbsOp : public OpConversionPattern<AtenAbsOp> {
+public:
+  using OpConversionPattern<AtenAbsOp>::OpConversionPattern;
+
+  LogicalResult
+  matchAndRewrite(AtenAbsOp op, OpAdaptor adaptor,
+                  ConversionPatternRewriter &rewriter) const override {
+    Value input = adaptor.getSelf();
+    RankedTensorType inputType = input.getType().dyn_cast<RankedTensorType>();
+    if (!inputType)
+      return rewriter.notifyMatchFailure(
+          op, "Only Ranked Tensor types are supported in TCP");
+    auto elementType = inputType.getElementType();
+    if (!elementType.isIntOrFloat())
+      return rewriter.notifyMatchFailure(
+          op, "Abs input tensor must have integer or floating-point datatype");
+
+    rewriter.replaceOpWithNewOp<tcp::AbsOp>(op, inputType, input);
+    return success();
+  }
+};
+
+template <typename AtenOpT, typename TcpOpT>
+class ConvertAtenTemplatedUnaryOp : public OpConversionPattern<AtenOpT> {
```

Comment:
nit: consider dropping `Templated` from the name (as it is self-explanatory)

Reply:
+1 to this